app/src/main/java/com/android/streetworkapp/ui/authentication/SignIn.kt

```diff
@@ -81,19 +99,102 @@ fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
+
+          Spacer(modifier = Modifier.height(26.dp).testTag("loginScreenSecondSpacer"))
+
+          IconAndTextRow(
+              imageVector = Icons.Filled.LocationOn,
+              contentDescription = "Location marker icon",
+              text = "Find nearby parks and events to participate in or create",
+              testName = "loginScreenFirstRow")
+
+          IconAndTextRow(
+              imageVector = Icons.AutoMirrored.Filled.DirectionsRun,
+              contentDescription = "Running person icon",
+              text = "Track your activities and learn new skills",
+              testName = "loginScreenSecondRow")
+
+          IconAndTextRow(
+              imageVector = Icons.Filled.ConnectWithoutContact,
+              contentDescription = "People connecting icon",
+              text = "Make new friends, train together and share activities",
+              testName = "loginScreenThirdRow")
 
-          Spacer(modifier = Modifier.height(64.dp).testTag("loginScreenSpacer"))
+          Spacer(modifier = Modifier.height(26.dp).testTag("loginScreenThirdSpacer"))
 
-          GoogleAuthButton(authService, context, launcher)
+          Box(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .height(96.dp)
+                      .background(ColorPalette.INTERACTION_COLOR_DARK)
+                      .testTag("loginScreenGoogleAuthButtonContainer"),
+              contentAlignment = Alignment.Center) {
+                GoogleAuthButton(authService, context, launcher)
+              }
         }
   }
 }
 
+/**
+ * A composable function that displays an icon and a text in a row with specified padding and
+ * alignment.
+ *
+ * @param imageVector The vector image to be used as the icon.
+ * @param contentDescription The content description for the icon, used for accessibility.
+ * @param text The text to be displayed next to the icon.
+ * @param testName The test tag name to be used for testing purposes.
+ */
+@Composable
+fun IconAndTextRow(
+    imageVector: ImageVector,
+    contentDescription: String,
+    text: String,
+    testName: String
+) {
+  Row(
+      modifier = Modifier.padding(horizontal = 25.dp, vertical = 8.dp).testTag(testName),
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.Center) {
+        Icon(
+            imageVector = imageVector,
+            contentDescription = contentDescription,
+            modifier = Modifier.weight(0.7f).aspectRatio(26.4f / 33f).testTag("${testName}Icon"),
+            tint = ColorPalette.INTERACTION_COLOR_DARK)
+        Spacer(modifier = Modifier.width(16.dp).testTag("${testName}Spacer"))
+        Text(
+            text = text,
+            style =
+                TextStyle(
+                    fontSize = 18.sp,
+                    lineHeight = 24.sp,
+                    fontWeight = FontWeight.W500,
+                    color = Color(0xFF000000)),
```

Comment:
Suggestion: This color could perhaps be added to the `ColorPalete`? (Or use the `Color.Black` , since i think it is also equal to 0xFF00000

Reply:
Indeed thanks for the comment! I changed with `Color.Black`, I think it's not so relevant to add it to the `ColorPalette` since it's a default `Color`.